src/features/mcda/readme.md

```diff
@@ -0,0 +1,78 @@
+## MCDA - Multi Criteria Decision Analysis
+
+MCDA is a basic tool to answer localization questions with GIS ("where better do or not to do something").
+Users have a need to perform a spatial analysis based on a variety of criteria.
+Previously Kontur allowed combining only 2 layers in bivariate-manager feature.
+In MCDA feature we can allow users to select as many layers as they need.
+
+## How to use
+
+To start using this feature user needs:
+
+- feature "bivariate_manager" turned on ("bivariate_manager": true)
```

Comment:
what `("bivariate_manager": true)` means?

Reply:
that feature "bivariate_manager" is turned on =)
it's when bivariate matrix is available

Reply:
removed this part to avoid confusion, I believe it's clear that it's about user's feature flag